klient/k8s/res/resource_test.go

```diff
@@ -80,6 +81,84 @@ func TestRes(t *testing.T) {
 	}
 }
 
+func TestResNoConfig(t *testing.T) {
+	defer func() {
+		if r := recover(); r == nil {
+			t.Error("expected panic while invoking Res without k8s config")
+		}
+	}()
+
+	Res(nil)
+}
+
+func TestResInvalidConfig(t *testing.T) {
+	defer func() {
+		if r := recover(); r == nil {
+			t.Error("expected panic while invoking Res with invalid k8s config")
+		}
+	}()
+
+	cfg := &rest.Config{
+		Host: "invalid-host",
+	}
+
+	Res(cfg)
```

Comment:
Not related to this specific PR but it would be good to rename this to use Resource and use the full term going forward (like in the package name too). "Res" can be interpreted as a lot of things and I don't think the shorthand form is that much easier than just "Resource"

Reply:
I agree on this! Will have a separate PR for this change.

Reply:
@andrewsykim I will be raising a separate PR for this change.